test_plans/oneapi/ext_oneapi_current_device.asciidoc

```diff
@@ -0,0 +1,35 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for ext oneapi current device
+
+This is a test plan for the device selector as described in SYCL 2020 revision 9 specification.
+
+Estimated development time is one day.
+
+== Testing scope
+
+=== Backend coverage
+
+All the tests described below are not backend-specific and are performed for any SYCL backend.
+
+=== Device coverage
+
+All tests construct a test device for which conformance is assessed. All of the tests described below are performed only on the default device that
+can be selected on the CTS command line.
+
+== Tests
+
+=== Feature test macro
+
+All of the tests should use `#ifdef SYCL_EXT_ONEAPI_CURRENT_DEVICE` so they can be skipped
+if feature is not supported.
+
+=== Functions `get_current_device` and `set_current_device`
+
+1. Get default device name from environment variable. Call `get_current_device()` but do not call `set_current_device`.
```

Comment:
You can get the default device just by default-constructing a `device` object:

```
void test() {
  sycl::device d;  // d is the default device
}
```

There is no need to use an environment variable.